# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webcodecs/moz.build
# Commit: eb17a0f3efdf
# Full Hash: eb17a0f3efdf67d5c7098c4ec5dc8ee57ea0956c
# Author: Chun-Min Chang <chun.m.chang@gmail.com>
# Date: 2023-06-29 15:31:45
# Regressor Bug: 1831451
# File Overlap Count: 2
# Description:
#   Bug 1831451 - Add a blank VideoDecoder interface r=webidl,padenot,edgar,smaug,saschanaz
#   
#   This patch introduces a VideoDecoder interface generated by running
#   `./mach webidl-example VideoDecoder` with necessary changes to make it
#   buildable.
# ==============================================================================

diff -r 7377bc0dc1f6 -r eb17a0f3efdf dom/media/webcodecs/moz.build
--- a/dom/media/webcodecs/moz.build	Thu Jun 29 03:13:27 2023 +0000
+++ b/dom/media/webcodecs/moz.build	Thu Jun 29 04:15:00 2023 +0000
@@ -9,6 +9,7 @@
 EXPORTS.mozilla.dom += [
     "EncodedVideoChunk.h",
     "VideoColorSpace.h",
+    "VideoDecoder.h",
     "VideoFrame.h",
     "WebCodecsUtils.h",
 ]
@@ -16,6 +17,7 @@
 UNIFIED_SOURCES += [
     "EncodedVideoChunk.cpp",
     "VideoColorSpace.cpp",
+    "VideoDecoder.cpp",
     "VideoFrame.cpp",
     "WebCodecsUtils.cpp",
 ]